src/style.css

```diff
@@ -0,0 +1,18 @@
+* {
+  margin: 0;
+}
+
+.block-1 {
+  height: 100px;
```

Comment:
you duplicate this rule, try to figure out how to write it only once

Reply:
Thank yo for review, i corrected mistake.